README.md

```diff
@@ -83,13 +67,13 @@ View the live demo [here](https://github.com/SAP/fundamental-ngx/tree/develop/do
 
 ## Tests
 
-Fundamental NGX makes use of Jasmine and Karma for its unit tests. 
+Fundamental NGX makes use of Jasmine and Karma for its unit tests.
 
 Run `ng test fundamental-ngx`. Append `--code-coverage` to generate code coverage documentation.
 
 ## Known Issues
 
-There are no known major issues. 
+There are no known major issues.
```

Comment:
consider following fundamental-react's lead here:

https://github.com/SAP/fundamental-react/blob/master/README.md#known-issues

Reply:
Jonathan sent out some comments on all our ReadMes at some point. We had it like you guys at some point, but we switched it. Here's the quote.

> Your **Known Issues** is not a pointer to the issues repository. It's a list of major issues that should be known in advance. If there are no major issues, just say "no major issues"

Reply:
The problem with that is, what constitutes a "major issue"?

between that, and the fact you would have to have PR just update your readme to display such information guarantee's it will be stale.

Reply:
@MattL75 @bcullman has a valid point and I think it is more transparent this way.

Reply:
updated.